### PR TITLE
Code style

### PR DESCRIPTION
**Pull request by Merk0ff**

### CODE REVIEW

**Review by kseniadumpling — APPROVED** [review]
Recommend test before merging

**Comment by Merk0ff** [comment]
> Recommend test before merging

tests 4 fags

File: main.py [hunk 1]
```diff
@@ -9,101 +11,105 @@
 
 def reform_attachments(attachments):
```
Comment (kseniadumpling) [inline main.py:15]: Add docstrings

File: main.py [hunk 2]
```diff
@@ -9,101 +11,105 @@
...
+                att = f'{type}{attachment[type]["owner_id"]}_{attachment[type]["id"]}'
+                if 'access_key' in attachment['type']:
+                    att += f'_{attachment[type]["access_key"]}'
+
+                attachment_arr.append(att)
+
     return attachment_arr
 
 
 def reform_forward_msg(forward_msgs):
```
Comment (kseniadumpling) [inline main.py:30]: Add docstrings

File: main.py [hunk 3]
```diff
@@ -9,101 +11,105 @@
...
-            #                   ,
-            #                                                forward_messages=reform_forward_msg(event.message.fwd_messages))
-
-            print ('Event handling ended')
-    except Exception as e:
-        print (e)
-        pass
-    print ('The End')
+
+def load_settings():
```
Comment (kseniadumpling) [inline main.py:41]: Add docstrings

**Review by kseniadumpling — COMMENTED** [review]
Some additional comments.
@Merk0ff, would you be so kind to add some docs?

**Comment by Merk0ff** [comment]
> Some additional comments.
> @Merk0ff, would you be so kind to add some docs?

Add self die

**Pull request merged**